src/test/app/ValidatorList_test.cpp

```diff
@@ -125,6 +125,15 @@ class ValidatorList_test : public beast::unit_test::suite
             keys.first, keys.second, makeSlice(data)));
     }
 
+    static hash_set<NodeID>
+    asNodeIDs(std::initializer_list<PublicKey> const& pks)
+    {
+        hash_set<NodeID> res;
```

Comment:
Nit: We know the final size of res. We could `res.reserve(pks.size());` here. Tho I'm less concerned about this sort of thing in test code.